bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1005,6 +1014,56 @@ lnd/clm2/surfdata_map/surfdata_ne120np4_78pfts_CMIP6_simyr1850_c170824.nc</fsurd
 <ndepmapalgo use_cn=".true."   hgrid="1x1_asphaltjungleNJ" >nn</ndepmapalgo>
 <ndepmapalgo use_cn=".true."   hgrid="5x5_amazon"          >nn</ndepmapalgo>
 
+<!--KO-->
+<!-- FAN Nitrogen (manure) deposition streams namelist defaults -->
+<stream_year_first_fan   sim_year="2000" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="2000" >2010</stream_year_last_fan>
+
+<stream_year_first_fan   sim_year="1850" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="1850" >2010</stream_year_last_fan>
+
+<stream_year_first_fan   sim_year="1000" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="1000" >2010</stream_year_last_fan>
+
+<stream_year_first_fan   sim_year="constant" sim_year_range="1000-1002" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="constant" sim_year_range="1000-1002" >2010</stream_year_last_fan>
+
+<stream_year_first_fan   sim_year="constant" sim_year_range="1000-1004" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="constant" sim_year_range="1000-1004" >2010</stream_year_last_fan>
+
+<stream_year_first_fan   sim_year="constant" sim_year_range="1850-2000" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="constant" sim_year_range="1850-2000" >2010</stream_year_last_fan>
+
+<stream_year_first_fan   sim_year="constant" sim_year_range="1850-2100" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="constant" sim_year_range="1850-2100" >2010</stream_year_last_fan>
+
+<stream_year_first_fan   sim_year="constant" sim_year_range="2000-2100" >2010</stream_year_first_fan>
+<stream_year_last_fan    sim_year="constant" sim_year_range="2000-2100" >2010</stream_year_last_fan>
+
+<stream_fldfilename_fan>nitrogen.nc</stream_fldfilename_fan>
+
+<fan_mapalgo                               >bilinear</fan_mapalgo>
+
+<fan_mapalgo   hgrid="1x1_brazil"          >nn</fan_mapalgo>
+<fan_mapalgo   hgrid="1x1_mexicocityMEX"   >nn</fan_mapalgo>
+<fan_mapalgo   hgrid="1x1_vancouverCAN"    >nn</fan_mapalgo>
+<fan_mapalgo   hgrid="1x1_urbanc_alpha"    >nn</fan_mapalgo>
+<fan_mapalgo   hgrid="1x1_camdenNJ"        >nn</fan_mapalgo>
+<fan_mapalgo   hgrid="1x1_asphaltjungleNJ" >nn</fan_mapalgo>
+<fan_mapalgo   hgrid="1x1_tropicAtl"       >nn</fan_mapalgo>
+<fan_mapalgo   hgrid="5x5_amazon"          >nn</fan_mapalgo>
+
```

Comment:
@ekluzek are these linked/ logical namelist changes typical elsewhere in the code?

Reply:
Yes, this same kind of thing is done for other streams.